package.json

```diff
@@ -51,6 +52,7 @@
     "@commitlint/cz-commitlint": "^19.2.0",
     "@commitlint/types": "^19.0.3",
     "@eslint-react/eslint-plugin": "^1.10.1",
+    "@eslint/config-inspector": "^0.5.4",
```

Comment:
If this dependency was added in this commit, it needs to be in a separate chore(deps)... The script additions should remain in this config commit, and if changed, a new config commit should be made.